### ISSUE TITLE
Minecraft Forge Support

### ISSUE THREAD

**Issue by cyanisaac** [zhuowei/Boardwalk#162]
Hello.

Your project has me very interested. It's really cool to know that you can play full Minecraft on a tablet... Even with lag it's cool.

I would really appreciate if there was some way that Forge could be installed. I want to run ComputerCraft, so I can program ComputerCraft on the go. I don't know how to get the files out of the universal .jar though, so I can't install it (since Android doesn't support jar files). If there was either a guide on how to extract Forge from the jar, or a tool to automate it, or emulation for the installer jar, it would help greatly.

Thanks for your awesome project and I hope you consider my suggestion.

**Comment by EnderBrine101** [comment]
This would be really cool, +1! We should have a way to drag mods into a mods folder from forge (Forge would be a seperate app) then it would wrap around boardwalk (Like blocklauncher does with minecraft pe) and install them that way. It should be done in a way magic launcher is in pc and how blocklauncher is because it doesnt modify and important files.

**Comment by EnderBrine101** [comment]
Good idea! Maybe we can run optifine but optifine doesnt support forge anyways. #166

**Comment by Mis012** [comment]
You have to root your device, and then you can acess 
/data/data/net.zhuoweizhang.boardwalk/app_working_dir/versions

**Comment by YellowXGaming** [comment]
I can't do it. I'm on Oppo R819 with ColorOS R819AS_11_131210. I have root but I can't access data/data/ file...
